### ISSUE TITLE
NC summary comparison dashboard not working

### ISSUE THREAD

**Issue by nichhk** [hackforla/311-data#1354]
### Overview
This dashboard is not working.  Chrome dev tools shows several errors like this (it's best to just navigate to the [dashboard](https://dash-reporting.tofn9kh9mlm7g.us-east-1.cs.amazonlightsail.com/dashboards/nc_summary_comparison) and see for yourself, though):
```
react-dom@16.v2_4_1m1661116854.14.0.min.js:125 Error: The children property of a component is a list of lists, instead of just a list. Check the component that has the following contents, and remove one of the levels of nesting: 
[
  {
    "props": {
      "children": {
        "props": {
          "id": "req_type_pie_chart",
          "style": {
            "height": "40vh",
            "width": "48.5vw"
          }
        },
        "type": "Graph",
        "namespace": "dash_core_components"
      },
      "style": {
        "display": "inline-block",
        "border": "0.5px black solid",
        "width": "48.5vw",
        "height": "40vh"
      }
    },
    "type": "Div",
    "namespace": "dash_html_components"
  }
]
```

### Action Items
- [ ] determine what's causing these errors
- [ ] fix the cause of the errors
### Resources/Instructions
[link to source code for this dashboard](https://github.com/hackforla/311-data/blob/dev/server/dash/dashboards/nc_summary_comparison.py)
- #1378

**Comment by joshuayhwu** [comment]
According to [this](https://stackoverflow.com/questions/62484218/how-to-fix-the-children-property-of-a-component-is-a-list-of-lists-instead-of) post it seems like I might have mistakenly included nested list in the layout. Let me check the code for req_type_pie_chart and get back to you.

**Comment by joshuayhwu** [comment]
I was taking a look at the code and there seems to be some issues with nested list in Dash based on the error. I used the nested divs primarily to align the visuals. As far as I can tell, the structure for the pie chart look something like:
```
html.Div(children=[html.Div(children=[html.Div(dcc.Graph(px.pie())), ...], ... ), ...])
``` 
[This](https://community.plotly.com/t/must-be-a-list-not-a-list-of-lists/56394) post also points out that it may actually be a missing / extra comma somewhere. Let me take a deeper look at the code and update.

**Comment by nichhk** [comment]
@joshuayhwu , if you're stuck, feel free to make a branch and push your current changes, and I can try debugging on my end.

**Comment by joshuayhwu** [comment]
@nichhk thanks so much. I have created PR #1355 but will continue to look at why the dynamic dropdown not showing up at all. My first thought was the Dash/Gunicorn issue causing callback functions to fail and not run at all. But it seems more like a react/dash issue based on the error message? 

Will continue look into this.

**Comment by nichhk** [comment]
Sure! Do you remember if at any point this dashboard was working correctly? If so, we can try to find the corresponding commit, and then check the diff between that commit and the current state to track down the bug(s).

**Comment by joshuayhwu** [comment]
fixed with #1387
